ebean-ddl-generator/src/test/resources/assert/ModelBuild_explicitSequencesTest/apply.sql

```diff
@@ -1,11 +1,11 @@
-create table persons (
-  id                            bigint generated by default as identity (start with 1000 increment by 40) not null,
-  surname                       varchar(64) not null,
-  name                          varchar(64) not null,
-  constraint pk_persons primary key (id)
+create table PERSONS (
+  ID                            bigint generated by default as identity (start with 1000 increment by 40) not null,
+  SURNAME                       varchar(64) not null,
+  NAME                          varchar(64) not null,
```

Comment:
@rbygrave due the removal of "lowerTableName", we will get uppercase names here.
I think, this is OK, because it is specified as `@Table(name = "PERSONS")` / ` @Column(name = "ID")`
If you want lower case names, specify them in the annotation